portal/views.py

```diff
@@ -236,3 +243,13 @@ def collection(request, slug, app_id=0):
     }
 
     return render(request, template, context=context)
+
+
+def events(request):
```

Comment:
```suggestion
def get_events(request):
```

Reply:
And it feels odd to accept all request verbs, but maybe it's fine. I'd restrict it to `GET` only.